arches/app/templates/base-root.htm

```diff
@@ -35,6 +35,16 @@
         gtag('config', '{{app_settings.GOOGLE_ANALYTICS_TRACKING_ID}}');
     </script>
     {% endif %}
+    <script>
+        const darkModeToggleState = localStorage.getItem("arches-dark");
```

Comment:
This class name is hard coded, because I don't see a way to read it dynamically from the DEFAULT_THEME js file. If project developers have a different use case, as a last resort, this template could be overridden.